src/main/java/com/bettercloud/vault/Vault.java

```diff
@@ -249,7 +249,7 @@ public Seal seal() {
         try {
             final RestResponse restResponse = new Rest()//NOPMD
                     .url(vaultConfig.getAddress() + "/v1/sys/mounts")
-                    .header("X-Vault-Token", vaultConfig.getToken())
+                    .optionalHeader("X-Vault-Token", vaultConfig.getToken())
```

Comment:
Huh.  It looks like another contributor added this `optionalHeader(...)` method back in February.  

If I thought about it at the time, I would have gone back and done all of this other cleanup to use it consistently (it's cleaner, and should have been like that to begin with).  Thanks for doing so.